content/release-notes/2023-01-17.md

```diff
@@ -0,0 +1,30 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Added support for several PostgreSQL extensions:
+`bloom`, `pgrowlocks`, `intagg`, `pgstattuple`,
+`earthdistance`, `postgis_topology`, `address_standardizer`, `address_standardizer_data_us`.
+
+https://github.com/neondatabase/neon/pull/3365/commits/a0ee306c740b4eb4a56aa9d422be7412ddf65b47
+
+https://github.com/neondatabase/neon/pull/3365/commits/dec875fee1683b88fb5f769b98bf432c07d34ed4
+
+
+- Proxy: Update SNI error message. Users now specify the endpoint ID when making a connection to Neon, not project ID. This change is reflected in the error message.
+
+https://github.com/neondatabase/neon/pull/3365/commits/424fd0bd63027cb325d87c4c07e70a88b2730aa1
+
+- Compute:
+Update client libraries and runtimes for Postgres libraries we test.
+- `pg8000` works with Neon now tada
+- `PostgresClientKit` still doesn't support SNI
+
+https://github.com/neondatabase/neon/pull/3365/commits/4c6b507472a985bf18e78adf0c3a5f4f922aa7ac
+
+
+- Compute: Add prefetch stat to EXPLAIN output.
+
+https://github.com/neondatabase/neon/pull/3365/commits/1983c4d4ade037a0ce66333fa54cc7d0707d936c
```

Comment:
@Daniel-ef 

We can create an additional article devoted to prefetch and what it means in our reality.

Reply:
```sql
EXPLAIN (PREFETCH) SELECT * FROM ANY_TABLE
```

As an example

```
neondb=> explain (prefetch) select 1;
                     QUERY PLAN
----------------------------------------------------
 Result  (cost=0.00..0.01 rows=1 width=4)
   Prefetch: hits=0 misses=0 expired=0 duplicates=0
(2 rows)
```

But it's better to create a table, generate data into it, and run the query that scans the table sequentially

Reply:
and a keynote is that the `PREFETCH` option in `EXPLAIN` is only available in Neon. It's not a standard PostgreSQL behavior.